app/models/question.rb

```diff
@@ -22,4 +22,8 @@ class Question < ActiveRecord::Base
   validates :number, presence: true, numericality: { only_integer: true }
 
   validates :question_type, presence: true, inclusion: {in: %w[short_answer multiple_choice true_false]}
+
+  def format_as_sms
+    return self.text + "\n" + self.response_choices.map{ |r| r.key + ") " + r.text}.join("\n")
```

Comment:
Don't need return, since last line of a method is automatically returned in ruby.